include/linker/kobject-text.ld

```diff
@@ -22,6 +24,7 @@
 #endif
 #endif
 
+#if defined(__GCC_LINKER_CMD__)
```

Comment:
It's better to use feature-flags.

e.g.

ifdef SUPPORTS_ASSERT

or ifdef SUPPORTS_MAX,

depending on what mwdt was struggling with.